DOC-2171: improvement documentation entry for TINY-9629 in the Release Notes (#2907)
tinymce/tinymce-docs: modules/ROOT/pages/6.7-release-notes.adoc

=== Updated **More** toolbar button tooltip text from *More...* to *Reveal or hide additional toolbar items*
// TINY-9629

> In previous versions of {productname}, a bug was discovered when using screen reader, but in particular https://www.freedomscientific.com/products/software/jaws[JAWS], within {productname}. The ellipsis icon representing the "more" options did not have a meaningful explanation for screen readers.
> 
> As a consequence, the screen reader would announce "More ellipsis" instead of providing a meaningful description that can be understood by users that rely on screen readers for accessibility.
> 
> To fix this, {productname} 6.7 updated the text label for the "more" options to "Reveal or hide more toolbar items" which now provides clear description for users that rely on this feature.

Reviewer: ```suggestion
In previous versions of {productname}, a bug was discovered when using screen readers, but in particular the https://www.freedomscientific.com/products/software/jaws[JAWS] screen reader, within {productname}. The ellipsis icon representing the **More** option in the {productname} toolbar did not have a meaningful explanation for screen readers.

As a consequence, screen readers would announce *More ellipsis* instead of providing a meaningful description that can be understood.

To fix this, {productname} 6.7 updated the text label for the *More* option to *Reveal or hide more toolbar items*; a clear description for users who rely on this feature.
```
Copy-edit suggestions.
